dali/operators/image/color/brightness_contrast.cc

```diff
@@ -25,33 +25,37 @@ using TheKernel = kernels::MultiplyAddCpu<Out, In, 3>;
 
 
 DALI_SCHEMA(BrightnessContrast)
-    .DocStr(R"code(Adjust the brightness and contrast of the image according to the formula::
+    .DocStr(R"code(Adjusts the brightness and contrast of the images based on the following
+formula::
 
   out = brightness_shift * output_range + brightness * (grey + contrast * (in - grey))
 
-where output_range is 1 for float outputs or the maximum positive value for integral types;
-grey denotes the value of 0.5 for float, 128 for `uint8`, 16384 for `int16`, etc.
+Where:
 
-Additionally, this operator can change the type of data.)code")
+-	The output_range is 1 for float outputs or the maximum positive value for integral types.
+-	Grey denotes the value of 0.5 for ``float``, 128 for ``uint8``, and 16384 for ``int16``, and so on.
+
+This operator can also change the type of data.)code")
     .NumInput(1)
     .NumOutput(1)
     .AddOptionalArg("brightness",
-                    "Brightness mutliplier; 1.0 is neutral.",
+                    "Brightness mutliplier, where 1.0 is neutral.",
```

Comment:
Do we really need to write that 1 is a neutral multiplier? I think it's rather common knowledge.

Reply:
I think we can remove

Reply:
Done